doc/source/release.rst

```diff
@@ -2,68 +2,68 @@
 Release Notes
 *************
 
-This is the list of changes to SciPy between each release. For full details,
+This is the list of changes to SciPy between each  For full details,-notes
```

Comment:
Unintentional deletion of release?
```suggestion
This is the list of changes to SciPy between each release. For full details,-notes
```

Reply:
Oops yes thanks! Let's put this in when we are done.